nestkernel/nest_time.h

```diff
@@ -387,7 +387,13 @@ class Time
   bool
   is_neg_inf() const
   {
-    return tics == LIM_NEG_INF.tics;
+    return tics <= LIM_NEG_INF.tics;
```

Comment:
@heplesser 
The assertion should be better performing than leave as is, and should only be worse than remove inequality when the assertion is activated with debug flags. Thus, it would be the best of both worlds.